test/images/agnhost/porter/porter.go

```diff
@@ -29,23 +29,24 @@ import (
 	"os"
 	"strings"
 
+	"github.com/ishidawataru/sctp"
 	"github.com/spf13/cobra"
 )
 
-const prefix = "SERVE_PORT_"
+const tcpPrefix = "SERVE_PORT_"
+const sctpPrefix = "SERVE_SCTP_PORT_"
 const tlsPrefix = "SERVE_TLS_PORT_"
 
 // CmdPorter is used by agnhost Cobra.
 var CmdPorter = &cobra.Command{
 	Use:   "porter",
 	Short: "Serves requested data on ports specified in ENV variables",
-	Long: `Serves requested data on ports specified in ENV variables. For example, if the environment variable "SERVE_PORT_9001" is set, then the subcommand will start serving on the port 9001.
-
-Additionally, if the environment variable "SERVE_TLS_PORT_9002" is set, then the subcommand will start a TLS server on that port.
-
-The included "localhost.crt" is a PEM-encoded TLS cert with SAN IPs "127.0.0.1" and "[::1]", expiring in January 2084, generated from "src/crypto/tls".
-
-To use a different cert/key, mount them into the pod and set the "CERT_FILE" and "KEY_FILE" environment variables to the desired paths.`,
```

Comment:
The information about the certificate / key is still useful and should still be kept, IMO.

Reply:
Agree